MAINTAINERS.yml

```diff
@@ -1841,6 +1838,489 @@ West:
+    - nashif
+  files:
+    - modules/Kconfig.xtensa
+  labels:
+    - manifest-hal_xtensa
+
+"West project: libmetal":
+  status: odd fixes
+  collaborators:
+    - carlocaione
+    - arnopo
+  files:
+    - modules/Kconfig.libmetal
+  labels:
+    - manifest-libmetal
+
+"West project: liblc3codec":
+  status: maintained
+  maintainers:
+    - Casper-Bonde-Bose
+  collaborators:
+    - thalley
+    - asbjornsabo
+  files:
+    - modules/liblc3codec/
+  labels:
+    - manifest-liblc3codec
+
+"West project: littlefs":
+  status: odd fixes
+  files:
+    - modules/littlefs/
+  labels:
+    - manifest-littlefs
+
+"West project: loramac-node":
+  status: maintained
+  maintainers:
+    - Mani-Sadhasivam
+  files:
+    - modules/loramac-node/
+  labels:
+    - manifest-loramac-node
+
+"West project: lvgl":
+  status: odd fixes
+  files:
+    - modules/Kconfig.lvgl
+  labels:
+    - manifest-lvgl
+
+"West project: lz4":
+  status: odd fixes
+  collaborators:
+    - Navin-Sankar
+  files:
+    - modules/lz4/
+  labels:
+    - manifest-lz4
+
+"West project: mbedtls":
+  status: maintained
+  maintainers:
+    - d3zd3z
+    - ceolin
+  files:
+    - modules/mbedtls/
+  labels:
+    - manifest-mbedtls
+
+"West project: mcuboot":
+  status: maintained
+  maintainers:
+    - d3zd3z
+  collaborators:
+    - nvlsianpu
+  files:
+    - modules/Kconfig.mcuboot_bootutil
+  labels:
+    - manifest-mcuboot
+
+"West project: mipi-sys-t":
+  status: odd fixes
+  maintainers:
+    - ...
+  collaborators:
+    - dcpleung
+  files:
+    - modules/Kconfig.syst
+  labels:
+    - manifest-mipi-sys-t
+
+"West project: nanopb":
+  status: odd fixes
+  collaborators:
+    - pdgendt
+  files:
+    - modules/nanopb/
+  labels:
+    - manifest-nanopb
+
+"West project: net-tools":
+  status: odd fixes
+  maintainers:
+    - rlubos
+  collaborators:
+    - jukkar
+  files: []
+  labels:
+    - manifest-net-tools
+
+"West project: nrf_hw_models":
+  status: maintained
+  maintainers:
+    - aescolar
+  files: []
+  labels:
+    - manifest-nrf_hw_models
+
+"West project: open-amp":
+  status: odd fixes
+  collaborators:
+    - carlocaione
+  files:
+    - modules/Kconfig.open-amp
+  labels:
+    - manifest-open-amp
+
+"West project: openthread":
+  status: maintained
+  maintainers:
+    - rlubos
+  collaborators:
+    - pdgendt
+  files: []
+  labels:
+    - manifest-openthread
+
+"West project: segger":
+  status: odd fixes
+  files: []
+  labels:
+    - manifest-segger
+
+"West project: sof":
+  status: maintained
+  maintainers:
+    - andyross
+    - nashif
+  files:
+    - modules/Kconfig.sof
+  labels:
+    - manifest-sof
+
+"West project: tflite-micro":
+  status: odd fixes
+  collaborators:
+    - laurenmurphyx64
+  files:
+    - modules/tflite-micro/
+  labels:
+    - manifest-tflite-micro
+
+"West project: tinycbor":
+  status: obsolete
+  files:
+    - modules/Kconfig.tinycbor
+  labels:
+    - manifest-tinycbor
+
+"West project: tinycrypt":
+  status: obsolete
```

Comment:
Hmm no, this is not obsolete. There are no viable plans or alternatives to replace it (yet).

Reply:
> Hmm no, this is not obsolete. There are no viable plans or alternatives to replace it (yet).

Yeah -- I wrote this patch before I wrote https://github.com/zephyrproject-rtos/zephyr/commit/89b07084d668fbc14da5ae1dbc7302b674c5633a. Without that patch, the only thing that fit areas without even a single collaborator was "obsolete". I will update these now that that patch is merged and we are OK with having "odd fixes" be a status for things without collaborators.